cxx/gendb.hh

```diff
@@ -34,6 +34,8 @@ class GenDB {
   GenDB(std::mt19937* prng, const PCleanSchema& schema,
         bool _only_final_emissions = false, bool _record_class_is_clean = true);
 
+  double logp_score() const;
```

Comment:
"  // Return the log probability of the data incorporated into the GenDB so far."